serving docs part 2 (#619)
ariga/atlas: doc/md/serving/intro.md

---
id: atlas-serving
slug: /serving/intro
title: Serving
---

## Introduction

> 

Author: Added

> Usually as a team using Atlas, you would like to deploy it as a service in your cloud data center and enjoy a centralized 

Reviewer: ```suggestion
When working as a team, it is ideal to deploy Atlas as a service in your cloud data center, allowing you to enjoy a  centralized 
```

instance.

> When using Atlas in such cases we recommend using persistent storage.

Reviewer: ```suggestion
In these cases, we recommend using persistent storage.
```

> When using Atlas in such cases we recommend using persistent storage.

Reviewer 2: ```suggestion
The Atlas Management UI is designed to be a long-running process. It is therefore recommended to run it against a persistent database that will make sure your team's schemas, database credentials, migration history and more are not lost in case you need to restart the service.
```

Author: Added

> Atlas encrypts all sensitive information using [AEAD](https://developers.google.com/tink/aead?hl=en) encryption with

Reviewer 2: Atlas uses [tink](https://developers.google.com/tink), a battle-tested encryption library created at Google, to encrypt all sensitive information. Following recommendations from the developers of tink, Atlas uses [AEAD](https://developers.google.com/tink/aead?hl=en) encryption with an *AES128_GCM* type key.

*AES128_GCM* Key type. Following recommendations from [tink](https://developers.google.com/tink) library that Atlas uses

> (and many other companies as well).

Reviewer 3: ```suggestion

```

In this section we will review two deployment and usage options:
1. Deploying on a VM/EC2 instance with persistent storage.
2. Deploying using an official Atlas Docker container.

### Persistent Storage

> When serving Atlas on a VM/EC2 you'll need to decide which persistent storage you would like Atlas to store all the

Reviewer: ```suggestion
When serving Atlas on a VM/EC2, you will need to decide on which persistent storage you would like Atlas to store the
```

configuration data at.

> For now, you can choose any or the supported Databases such as:

Reviewer: ```suggestion
For now, you can choose any of the supported databases:
```
